src/proc.ts

```diff
@@ -12,9 +12,9 @@ import rollbar from './rollbar';
 import * as util from './util';
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
-import { ExecutionResult } from './api';
+import { ExecutionResult } from '@cmt/cmakeProject';
 import { Environment, EnvironmentUtils } from './environmentVariables';
-export { ExecutionResult } from './api';
+export { ExecutionResult } from '@cmt/cmakeProject';
```

Comment:
It feels like maybe `ExecutionResult` belongs in this module instead of `cmakeProject`.

Reply:
I moved `ExecutionResult` to `proc.ts` and deleted `ExecutionOptions`. `proc.ts` had and existing different definition of `ExecutionOptions`, and it looks like the `cmakeProject.ts` version was only used by the now-removed API.